src/style.css

```diff
@@ -0,0 +1,49 @@
+html {
+  --star_acive: url(images/star-active.svg);
+  --star: url(images/star.svg);
+}
+
+div,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  /* justify-content: center; */
+}
+
+.stars__star {
+  background-position: center;
+  object-fit: cover;
+  display: flex;
+  background-repeat: no-repeat ;
+  width: 16px;
+  height: 16px;
+  background-image: var(--star);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :nth-child(-n + 1) {
+  background-image: var(--star_acive);
+}
+
+.stars--2 > :nth-child(-n + 2) {
+  background-image: var(--star_acive);
+}
+
+.stars--3 > :nth-child(-n + 3) {
+  background-image: var(--star_acive);
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  background-image: var(--star_acive);
+}
+
+.stars--5 > * {
+  background-image: var(--star_acive);
+}
```

Comment:
Checklist:
2. [CODE STYLE] - If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future
![image](https://github.com/mate-academy/layout_stars/assets/63243136/f7ae6d7e-3a9d-44e1-98f0-9d5e999c3ba3)